app/assets/javascripts/discourse/lib/user-search.js.es6

```diff
@@ -44,7 +44,7 @@ function organizeResults(r, options) {
   if (r === CANCELLED_STATUS) { return r; }
 
   var exclude = options.exclude || [],
-      limit = options.limit || 5,
+      limit = options.limit || 6,
```

Comment:
Well, according to the previous logic, the intention is 6. I'll change that to 5.